Project 1/styles.css

```diff
@@ -0,0 +1,56 @@
+body {
+
+    padding-left: 120px;
+    padding-right: 120px;
+    max-width: 700px;
+    min-width: 500px;
+    background-color: rgb(113, 17, 17);
+    margin: 0 auto;
+}
+
+header {
+    background-color: #f7f7f7;
+    max-width: 300px;
+    min-width: 250px;
+    padding: 30px;
+    margin: 30px auto;
+    border: 5px solid rgb(255, 189, 67);
+}
+
+h1,
+h2 {
+    font-family: 'Spectral', serif;
+    text-align: center;
+}
+
+h1 {
+    font-size: 50px;
+    font-weight: 500px;
+    margin: 0;
+    color: rgb(0, 0, 0);
+    line-height: 50px;
+}
+
+h2 {
+    font-size: 13px;
+    line-height: 10px;
+}
+
+
+
+main {
```

Comment:
The `header` and `main` have some shared styles around their colors, padding and borders -- if we were going to continue this project for another week, I'd challenge you to think of a way to add a shared class to these elements to combine the bits they share.